airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -34,13 +34,15 @@ class GoogleCloudBaseHook(BaseHook):
     The class also contains some miscellaneous helper functions.
 
     All hook derived from this base hook use the 'Google Cloud Platform' connection
-    type. Two ways of authentication are supported:
+    type. Three ways of authentication are supported:
 
     Default credentials: Only specify 'Project Id'. Then you need to have executed
     ``gcloud auth`` on the Airflow worker machine.
 
     JSON key file: Specify 'Project Id', 'Key Path' and 'Scope'.
 
+    JSON key dict: Specify 'Project Id' and 'Scope'.
```

Comment:
Should this be:

    JSON key dict: Specify 'Project Id', 'Key Dict' and 'Scope'.

?